src/builtins/core/duration/tests.rs

```diff
@@ -452,3 +455,17 @@ fn total_full_numeric_precision() {
     let d = Duration::new(0, 0, 0, 0, 0, 0, 0, MAX_SAFE_INTEGER + 1, 1999, 0).unwrap();
     assert_eq!(d.total(Unit::Millisecond, None).unwrap(), 9007199254740994.);
 }
+
+
+#[test]
+fn round_zero_duration() {
```

Comment:
Ahhh, to use the `round` method over `round_with_provider`, the test would need to be in src/builtins/compiled/duration/tests` or the test can use the provider directly.

Reply:
Done in [9c76631](https://github.com/boa-dev/temporal/pull/647/commits/9c766311db3943abc4ec19771a6ae6150db91028)